prow/config/inrepoconfig.go

```diff
@@ -41,6 +43,22 @@ const (
 	inRepoConfigDirName  = ".prow"
 )
 
+var inrepoconfigMetrics = struct {
+	gitCloneDuration *prometheus.HistogramVec
+}{
+	gitCloneDuration: prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Name:    "git_execution_duration",
```

Comment:
I think this naming is confusing. `gitCloneDuration`, `git_execution_duration`, and the help text all seem like they mean different things. It would be nice to align those.
I suggested below that we track all git operations with this metric so `git_execution_duration` sounds pretty good, but maybe we could make it more specific and call it `inrepo_config_git_duration` or something like that.

Reply:
Oops, I was supposed to change the name away from `execution` language (because we're not measuring things at that super-low git executor level). Will update.